.pre-commit-config.yaml

```diff
@@ -121,6 +121,15 @@ repos:
     - cheroot/
     pass_filenames: false
 
+- repo: local
```

Comment:
This means that `pre-commit` doesn't provision its own virtualenv and uses the current one. In our case, that env we run `pre-commit` in is managed by `tox` so `stubtest` should be installed there. And for the `pre-commit.ci`, this should be skipped via the `ci:` setting in this config file because they run it directly and won't have it pre-installed there.

Reply:
Can you please support me to achieve this?

Reply:
1) In this file, between `---` and `repo:`, add the following:
    ```
    ci:
      skip:
      - stubtest
    ```
    It will make sure that this hook will not run in pre-commit.ci
2) For the `tox -e pre-commit` invocation, we actually want to run it. So go to `tox.ini`. Find the `[testenv:pre-commit]` section. Find the `deps` setting and add `stubtest` to the list, on a new line.